<!DOCTYPE html>
<html lang="en">
  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="assets/ico/favicon.png">
	

    <title>Jayendra Shinde Résumé</title>

<!-- Open Sans Google font -->
		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400">
		<!-- Raleway Google font -->
		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:700">
		<!-- Favicons -->
		
		<link rel="apple-touch-icon" sizes="60x60" href="img/favico/apple-icon-60x60.png">
		<!-- Ionicon fonts css file -->
		<link rel="stylesheet" href="assets/css/ionicons.min.css">
		<!-- magnific popup css file -->
		<link rel="stylesheet" href="assets/css/magnific-popup.css">
		<!-- Animate.css file -->
		<link rel="stylesheet" href="assets/css/animate.css">
		<!-- Owl carousel css file -->
		<link rel="stylesheet" href="assets/css/owl.carousel.css">
		<!-- Bootstrap file -->
		<link rel="stylesheet" href="assets/css/bootstrap.min.css">
		<!-- Custom styles css file -->
		<link rel="stylesheet" href="assets/css/style.css">

    <!-- Bootstrap core CSS -->
    <link href="assets/css/bootstrap.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="assets/css/main.css" rel="stylesheet">
    
	<link rel="stylesheet" href="assets/css/font-awesome.min.css">
    
    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
    
    <script src="assets/js/jquery.min.js"></script>
    <script type="text/javascript" src="assets/js/smoothscroll.js"></script> 
    <script src="assets/js/Chart.js"></script>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="assets/js/html5shiv.js"></script>
      <script src="assets/js/respond.min.js"></script>
    <![endif]-->
  </head>

  <body data-spy="scroll" data-offset="0" data-target="#nav">

	<canvas height="300" style="background: rgb(55, 58, 66)"></canvas>
  <script src="js/d3.v4.0.0-alpha.33.min.js"></script>
  <script>

(function() {
			var 
				// Obtain a reference to the canvas element
				// using its id.
				htmlCanvas = document.querySelector('canvas'),
			
        // Obtain a graphics context on the
        // canvas element for drawing.
        context = htmlCanvas.getContext('2d');
 
			// Start listening to resize events and
			// draw canvas.
			initialize();
 
			function initialize() {
				// Register an event listener to
				// call the resizeCanvas() function each time 
				// the window is resized.
				window.addEventListener('resize', resizeCanvas, false);
				
				// Draw canvas border for the first time.
				resizeCanvas();
			}
								
			function redraw() {
        var width = htmlCanvas.width,
          height = htmlCanvas.height;
				var n = 400,
        pi = Math.PI,
        tau = 2 * pi;

        var nodes = d3.range(n).map(function () {
          var r = Math.random() * width / 3,
            a = Math.random() * tau,
            x = width / 2 + r * Math.cos(a),
            y = height / 2 + r * Math.sin(a);
          return {
            x: x,
            y: y,
            vx: (height / 2 - y) * 0.006,
            vy: (x - width / 2) * 0.006
          };
        });

        var force = d3.forceSimulation(nodes)
          .drag(0)
          .alphaDecay(0)
          .force("charge", d3.forceManyBody().strength(0.02))
          .force("center", d3.forceCenter(width / 2, height / 2))
          .on("tick", ticked);

        var stroke = d3.scaleLinear()
          .domain([0, 5])
          .range(["rgb(205, 101, 83)", "rgb(150, 192, 206)"]);

        function ticked() {
          context.clearRect(0, 0, width, height);
          context.lineWidth = 5;
          context.lineCap = "circle";

          for (var i = 0, node, vx, vy; i < n; ++i) {
            node = nodes[i];
            context.beginPath();
            context.moveTo(node.x, node.y);
            context.lineTo(node.x + node.vx * 3, node.y + node.vy * 3);
            context.strokeStyle = stroke(node.vx * node.vx + node.vy * node.vy);
            context.stroke();
          }
        }
			}
		
			// Runs each time the DOM window resize event fires.
			// Resets the canvas dimensions to match window,
			// then draws the new borders accordingly.
			function resizeCanvas() {
				htmlCanvas.width = window.innerWidth;
				htmlCanvas.height = 150;
				redraw();
			}
		
		})();

  </script>

	<!-- Start About Me Section -->
		
		<div class="about-me clearfix" id="about-me">
			<div class="container">
				<div class="row">
					<!-- Profile Pic -->
					<div class="my-pic center col-sm-4 col-xs-12 wow fadeInUp" data-wow-duration="0.5s" data-wow-offset="200">
						<img src="face1.png" alt="my profile">
					</div>
					<div class="introduction col-sm-8 col-xs-12 wow fadeInUp" data-wow-duration="0.5s" data-wow-offset="200">
						<h2 class="h2">Jayendra Shinde</h2>
						<div class="heading-line"></div>
						<p class="introduce"></p>
						<p class="intro"> I’m a postdoctoral researcher at  <a href="https://bmir.stanford.edu/"> Stanford Center for Biomedical Informatics Research (BMIR) </a> in the  <a href="http://med.stanford.edu/gevaertlab.html/">Gevaert Lab</a>.  Before joining Stanford University, I was a PhD student in Computational Biology, working under the supervision of Prof. Jessica Zucman-Rossi and Dr. Eric Letouze at <a href="http://zucmanlab.com">INSERM UMR1162</a> in Paris.</p> </br>
						<p class="intro"> Major interest areas:</p> 
						<p class="intro"> Cancer Genomics, Personalized Medicine, Medical Image Analysis, Healthcare, Data Science, Data Visualization, Health Informatics and Machine Learning </p> 
					</div>
				</div>

<hr>
<div class="title">
					<h2 class="h2">Education</h2>
					<div class="heading-line"></div>
					<p class="main-para">
	<section id="resume" name="resume"></section>
	<!--EDUCATION DESCRIPTION -->
	<div class="container desc">
		<div class="row">
				<div class="col-lg-2 col-lg-offset-1">
					<h4></h4>
				</div>
				<div class="col-lg-6">
					<p><t>PhD in Computational Biology</t><br/>
						*INSERM UMR 1162 <br/>
						Génomique Fonctionnelle des Tumeurs Solides,<br/>
						Equipe Labellisée Ligue Contre le Cancer, Institut Universitaire d'Hematologie (IUH), Paris, France.<br/>
						*Université Paris Diderot, Paris, France.<br/>
						<i>October 2014</i>
					</p>
				</div>
				<div class="col-lg-3">
					<p><strong>Graduation</strong><br/>
					<sm>November 2017</sm>
					</p>
				</div>
	
				<div class="col-lg-6 col-lg-offset-3">
					<p><t>Integrated Masters in Biotechnology</t><br/>
						Institute of Bioinformatics and Biotechnology (IBB)<br/>
						University of Pune <br/>
						Pune, India <br/>
						<i>July 2009 </i>
					</p>
				</div>
				<div class="col-lg-3">
					<p><sm>May 2014</sm></p>
				</div>
	
		</div><!--/.row -->
		<br>
		<hr>
	</div><!--/.container -->
	</div>
	
	

				<!-- Title -->
				<div class="title">
					<h2 class="h2">Research</h2>
					<section id="research" name="resume"></section>
					<div class="heading-line"></div>
					<p class="main-para">Human cancers develop due to accumulation of somatic mutations which result from diverse processes including exposure to environmental mutagens such as UV light, tobacco smoke as well as endogenous DNA damaging processes like replication errors, spontaneous or enzymatic conversions. I develop computational methods in order to investigate these various patterns of mutational signatures that arise in cancer genomes. The major focus of my research is based on understanding the interaction of intracellular processes with diverse mutational processes and ultimately their contribution towards cancer evolution. Studying these complex patterns of mutational heterogeneity might be useful in effective patient stratification. My career goals are to use cancer metadata and omics data, with computational modeling, and big data analytics in order to improve cancer care.</p>
				<p></p>
<p></p>

					<div id="myCarousel" class="carousel slide" data-ride="carousel">
					  <!-- Indicators -->
					  <ol class="carousel-indicators">
					    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
					    <li data-target="#myCarousel" data-slide-to="1"></li>
					    <li data-target="#myCarousel" data-slide-to="2"></li>
					  </ol>

					  <!-- Wrapper for slides -->
					  <div class="carousel-inner">
					    <div class="item active">
					      <img src="images/img1.jpg" alt="img1">
					    </div>

					    <div class="item">
					      <img src="images/img2.jpg" alt="img2">
					    </div>

					    <div class="item">
					      <img src="images/img3.jpg" alt="img3">
					    </div>

					    <div class="item">
					      <img src="images/img4.jpg" alt="img4">
					    </div>
					  </div>



				</div>
				<p></p>
				<hr>
<p></p>
<p></p>
<div class="title">
					<h2 class="h2">Publications</h2>
					<section id="research" name="resume"></section>
					<div class="heading-line"></div>
					
					<p class="main-para">1. Bayard, Q., Meunier, L., Peneau, C., Renault, V., <b>Shinde, J.</b>, Nault, J.-C., … Letouzé, E. (2018). Cyclin A2/E1 activation defines a hepatocellular carcinoma subclass with a rearrangement signature of replication stress. Nature Communications, 9(1), 5235. https://doi.org/10.1038/s41467-018-07552-9</p><a href="hhttps://www.nature.com/articles/s41467-018-07552-9#disqus_thread">[LINK]</a><a href="papers/QB-etal.pdf">[PDF]</a><p></p>

					<p class="main-para">2. <b>Shinde, J.</b>, Bayard, Q., Imbeaud, S., Hirsch, T.Z., Liu, F., Renault, V., Zucman-Rossi, J. and Letouzé, E. (2018) 'Palimpsest: an R package for studying mutational and structural variant signatures along clonal evolution in cancer', Bioinformatics.</p><a href="https://github.com/FunGeST/Palimpsest">[GitHub]</a><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty388/4996591?redirectedFrom=fulltext">[LINK]</a><a href="papers/1.pdf">[PDF]</a><p></p>
					<p></p>
					<p class="main-para">3. Letouzé, E.*, <b>Shinde, J.*</b>, Renault, V., Cauchy, G., Blanc, JF., Tubacher,E., Bayard, Q., Bacq, D.,
					Meyer,. V., Semhoun, J., Bioulac-Sage, P., Prévôt, S., Azoulay, D., Paradis, V., Imbeaud, S.,
					Deleuze, JF and Zucman-Rossi, J. (2017) Letouzé, E., Shinde, J., Renault, V., Couchy, G., Blanc, J.-F., Tubacher, E., Bayard, Q., Bacq, D., Meyer, V., Semhoun, J., Bioulac-Sage, P., Prévôt, S., Azoulay, D., Paradis, V., Imbeaud, S., Deleuze, J.-F. and Zucman-Rossi, J. (2017) ‘Mutational signatures reveal the dynamic interplay of risk factors and cellular processes during liver tumorigenesis’, Nature Communications. Nature Publishing Group, 8(1), p. 1315. doi: 10.1038/s41467-017-01358-x</p>
					<a href="https://www.nature.com/articles/s41467-017-01358-x">[LINK]</a><a href="papers/2.pdf">[PDF]</a><p></p>

					<p class="main-para">4. Pilati, C.*,<b>Shinde, J.*</b>, Alexandrov, L. B.*, Assié, G., André, T., Hélias-Rodzewicz, Z., Ducoudray, R.,
					Le Corre, D., Zucman-Rossi, J., Emile, J.-F., Bertherat, J., Letouzé, E. and Laurent-Puig, P. (2017)
					‘Mutational signature analysis identifies MUTYH deficiency in colorectal cancers and adrenocortical
					carcinomas’, The Journal of Pathology, 242(1), pp. 10–15. doi: 10.1002/path.4880</p><a href="http://onlinelibrary.wiley.com.gate2.inist.fr/doi/10.1002/path.4880/abstract">[LINK]</a><a href="papers/3.pdf">[PDF]</a>

					<p></p>
					<p class="main-para">5. Schulze, K.*, Imbeaud, S.*, Letouzé, E.*, Alexandrov, L. B., Calderaro, J., Rebouissou, S., Couchy,
					G., Meiller, C., <b>Shinde, J.</b>, Soysouvanh, F., Calatayud, A.-L., Pinyol, R., Pelletier, L., Balabaud, C.,
					Laurent, A., Blanc, J.-F., Mazzaferro, V., Calvo, F., Villanueva, A., Nault, J.-C., Bioulac-Sage, P.,
					Stratton, M. R., Llovet, J. M. and Zucman-Rossi, J. (2015) ‘Exome sequencing of hepatocellular
					carcinomas identifies new mutational signatures and potential therapeutic targets’, Nature Genetics.
					Nature Publishing Group, 47(5), pp. 505–511. doi: 10.1038/ng.3252</p><a href="http://www.nature.com/ng/journal/v47/n5/full/ng.3252.html">[LINK]</a><a href="papers/4.pdf">[PDF]</a>
					<p></p>
					<p></p>		
					<b>* Authors contributed equally to this work.</b>			
					<p></p>

<div class="title">
	<h2 class="h2">Contact</h2>
	<div class="heading-line"></div>
	<p class="main-para">
	<section id="resume" name="resume"></section>
				<div class="col-lg-6">
					<br/>
					<br/>
					<p><t>Email</t><br/>
						jayendra at stanford dot edu<br/>
					</p>
					<p><t>Address</t><br/>
						Stanford Center for Biomedical Informatics Research (BMIR) <br/>
						Department of Medicine, Stanford University <br/>
						1265 Welch Rd, Stanford, CA 94305-5479 <br/>

					</p>
			
				</div>
				<div class="col-lg-2">
					<p><sm> </sm></p>
						
						<a href="https://github.com/jayendrashinde91"><img src="./logos/github.svg" width="56px"></br> 
							</br>
						<a href="https://fr.linkedin.com/in/jayendra-shinde-5809a074"><img src="./logos/linkedin.png" width="62px"></br>
							</br>
						<a href="https://twitter.com/jayendradotcom?lang=en"><img src="./logos/twitter.png" width="56px"></a></br>

					</p>
				</div>

</div>
				<div class="col-lg-4">
					<a class="twitter-timeline" data-tweet-limit="1" data-theme="light" data-link-color="#CD6553" href="https://twitter.com/jayendradotcom">Tweets by Jayendra</a> 
        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <div class="showmobile" style="margin-top: -20px; margin-bottom: 10px;">
          <hr style="margin-bottom: 5px;">You can follow me on Twitter at
          <a href="https://twitter.com/jayendradotcom">@jayendradotcom</a>.

			</div><!--/.row -->
		</div><!--/.container -->
	</div><!--/ #footer -->
      </div>


<footer class="footer-global">

	</div>
			</div><!--/.row -->
		</div><!--/.container -->
	</div><!--/ #footer -->
	<div id="c">
		<div class="container">
		<p>Copyright &copy; Jayendra Shinde 2018</p>
		<div id="footer-navi">
    <ul>
        <li class="footer-item">
      <a class="logo logo-stanford" href="http://www.stanford.edu" title="">Stanford University</a>
    </li>
    <li class="footer-item">
      <a class="logo logo-stanford-som" href="http://med.stanford.edu" title="">Stanford School of Medicine</a>
    </li>
    <li class="footer-item">
      <a class="logo logo-stanford-hospital" href="http://stanfordhealthcare.org" title="">Stanford Health Care</a>
    </li>
    
    </ul>
</div>
		</div>
	</div>
	</footer>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="assets/js/bootstrap.js"></script>
    <!-- Including jQuery file -->
		<script src="assets/js/jquery-1.12.1.min.js"></script>
		<!-- Including jQuery.appear file -->
		<script src="assets/js/jquery.appear.js"></script>
		<!-- Bootstrap js file -->
		<script src="assets/js/bootstrap.min.js"></script>
		<!-- easy pie chart -->
		<script src="assets/js/jquery.easypiechart.min.js"></script>
		<!-- NiceScroll plugin -->
		<script src="assets/js/jquery.nicescroll.min.js"></script>
		<!-- animate numbers plugin -->
		<script src="assets/js/jquery.animateNumber.min.js"></script>
		<!-- Horizontal bar chart -->
		<script src="assets/js/bars.js"></script>
		<!-- mixitup plugin -->
		<script src="assets/js/jquery.mixitup.min.js"></script>
		<!-- animated background header -->
		<script src="assets/js/TweenLite.min.js"></script>
		<!-- magnific popup js file -->
		<script src="assets/js/magnific-popup.js"></script>
		<!-- form validation file -->
		<script src="assets/js/jqBootstrapValidation.js"></script>
		<!-- Owl carousel js file -->
		<script src="assets/js/owl.carousel.min.js"></script>
		<!-- Google Map -->
		<script src='https://maps.googleapis.com/maps/api/js?v=3.exp'></script>
		<!-- Custom file  -->
		<script src="assets/js/plugins.js"></script>
		<!-- Wow.js file -->
		<script src="assets/js/wow.min.js"></script>

  </body>
</html>
